src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs

```diff
@@ -45,22 +45,23 @@ public IEnumerable<PackageVerifierIssue> Validate(PackageAnalysisContext context
                             if (AssemblyHelpers.IsAssemblyManaged(assemblyPath))
```

Comment:
What happens when you call `ReadAssembly` on a non-managed assembly? I'd think it throws something useful.